scylla-cql/src/types/serialize/value.rs

```diff
@@ -223,7 +223,13 @@ impl SerializeCql for f64 {
 }
 impl SerializeCql for Uuid {
     impl_serialize_via_writer!(|me, typ, writer| {
-        exact_type_check!(typ, Uuid, Timeuuid);
+        exact_type_check!(typ, Uuid);
```

Comment:
Just a note: this change means that PR is API-breaking I think, we need to mark it as such so that it's properly described in release notes.